pcdsdevices/epics_motor.py

```diff
@@ -191,7 +155,8 @@ def check_value(self, value):
         # Find the soft limit values from EPICS records and check that this
         # command will be accepted by the motor
         if any(self.limits):
-            if not (self.low_limit <= value <= self.high_limit):
+            if not (self._get_epics_limits()[0] <= value
```

Comment:
I was not sure here if I should have changed this. The `low_limit` and `high_limit` work as well (at least in some situations), because these are inherited from the ophyd Component (I think?)

Reply:
For reference on inherited `low_limit` and `high_limit` attributes: https://github.com/bluesky/ophyd/blob/c91317fb21506d6acc2900e392e667aeb2684abc/ophyd/positioner.py#L142-L148

Is the concern here just the case where limits are uninitialized or invalid? It may be easiest to just wrap the `_get_epics_limits` check into an override of the `limits` attribute in that case. Then all instances of `self.limits`, `self.low_limit`, `self.high_limit` work as expected.

Reply:
Overrode the `limits` attribute in 175f3e1 - and changed to using `limits` in 44f9852